src/main/java/org/broadinstitute/hellbender/utils/pca/PCA.java

```diff
@@ -51,9 +77,14 @@
      *
      * @throws IllegalArgumentException if either {@code dataMatrix} or {@code svdFactory} is {@code null}.
      */
-    public static PCA createPCA(final RealMatrix dataMatrix, final Function<RealMatrix, SVD> svdFactory) {
+    public static PCA createPCA(final List<String> variables, final List<String> samples, final RealMatrix dataMatrix, final Function<RealMatrix, SVD> svdFactory) {
```

Comment:
going forward, do we want to always require variable and sample names when doing PCA?  perhaps give the option to createPCA with no variable/sample names, and only check for names when writing to HDF5?